tests/FiberTraces.py

```diff
@@ -687,7 +687,22 @@ def testFiberTraceSetFunctions(self):
             self.assertEqual(spectra.size(), ftsComb.size())
             for i in range(spectra.size()):
                 self.assertEqual(spectra.getSpectrum(i).getLength(), ftsComb.getFiberTrace(i).getHeight())
-            
+
+    def testAddFiberTraceToCcdImage(self):
+        fts = drpStella.findAndTraceAperturesF(self.flat.getMaskedImage(), self.ftffc)
+        origMaskedImage = self.flat.getMaskedImage()
+        imageShape = origMaskedImage.getImage().getArray().shape
+        sumImage = afwImage.ImageF(np.zeros(shape=imageShape, dtype=np.float32))
+        ftsOrig = fts
+        for iFt in range(fts.size()):
+            ft = fts.getFiberTrace(iFt)
+            drpStella.addFiberTraceToCcdImage(ft, ft.getImage(), sumImage)
+        self.assertTrue(fts.createTraces(afwImage.makeMaskedImage(sumImage)))
```

Comment:
Yes, `FiberTraceSet.createTraces` does change the `FiberTraceSet`. I offered this function so we can simply extract the FiberTraces from a new image without the need to copy all the object variables. I added a comment to `FiberTraces.h` to make this clear